CHANGELOG.md

```diff
@@ -33,7 +34,8 @@ Release of genomic-medicine-sweden/tomte, created with the [nf-core](https://nf-
 - Shortened name of DROP output files [#79](https://github.com/genomic-medicine-sweden/tomte/pull/79)
 - Merging of vcfs has been moved to after bootstrapAnn [#81](https://github.com/genomic-medicine-sweden/tomte/pull/81)
 - Substituted bgzip and tabix modules by bgzip_tabix module [#85](https://github.com/genomic-medicine-sweden/tomte/pull/85)
-- Updated module input channels in the GATK variant calling subworkflow [#89](https://github.com/genomic-medicine-sweden/tomte/pull/85)
+- Updated module input channels in the GATK variant calling subworkflow [#89](https://github.com/genomic-medicine-sweden/tomte/pull/89)
+- Subsample and downsample switches [#97](https://github.com/genomic-medicine-sweden/tomte/pull/97)
```

Comment:
Could you make a new heading in the CHANGELOG and add these fixes to that one as these fixes aren't part of the Rudolph release

Reply:
That's true!